PhysicsTools/PatAlgos/python/recoLayer0/bTagging_cff.py

```diff
@@ -42,7 +42,9 @@
   , 'pfDeepFlavourTagInfos'
   , 'pfNegativeDeepFlavourTagInfos'
     # ParticleTransformerAK4 tag infos
-  , 'pfParticleTransformerAK4TagInfos'
+  # , 'pfParticleTransformerAK4TagInfos'
```

Comment:
Please keep the relevant ones for `pfParticleTransformerAK4` in this file, so it can still be run if needed.